lib/isIdentityCard.js

```diff
@@ -52,6 +52,11 @@ var validators = {
     });
     return c === 0;
   },
+  IT: function IT(str) {
+    if (str.length !== 9) return false;
+    if (str === 'CA00000AA') return false; // https://it.wikipedia.org/wiki/Carta_d%27identit%C3%A0_elettronica_italiana
+    return str.search(/C[A-Z]\d{5}[A-Z]{2}/is) > -1
+  },
```

Comment:
Please add the code in `src/lib/isIdentityCard.js` for the locale to be added. This file will be autogenerated.